test-it/karma/utilities/TestHelpers.HttpParser.js

```diff
@@ -35,7 +35,7 @@
     var parseUrlElement = document.createElement('a');
 
     var parseUrl = function (url) {
-        // Add and remove the element to workaround IE https://connect.microsoft.com/IE/Feedback/Details/1002846
+        // Add and remove the element to workaround behavior in legacy browsers, no longer required
```

Comment:
Could you add a vireo issue to remove the instances where you said a section is no longer required.

Reply:
I don't think we need a specific issue to address these. None of the spots where I made the change are incorrect, just add extra checks, etc, but in the future if someone is curious why some behavior is there it is documented.